iceoryx_posh/include/iceoryx_posh/popo/subscriber_options.hpp

```diff
@@ -33,7 +33,7 @@ struct SubscriberOptions
 {
     /// @brief The size of the receiver queue where chunks are stored before they are passed to the user
     /// @attention Depending on the underlying queue there can be a different overflow behavior
-    uint64_t queueCapacity{SubscriberPortData::ChunkQueueData_t::MAX_CAPACITY};
+    uint64_t queueCapacity{iox::MAX_SUBSCRIBER_QUEUE_CAPACITY};
```

Comment:
I'm not quite sure if this is the way to go. I know that some stuff needed to be refactored. Maybe further down the PR it's becomes clear.

Reply:
I'm still not sure if this is a good idea or what consequences this will have when not `SubscriberPortData::ChunkQueueData_t::MAX_CAPACITY` is used. It might be better to do it like for the RPC feature where a `client_server_port_types.hpp` was created to prevent cyclic dependencies

Reply:
Because when someone sets a different value to `SubscriberPortData::ChunkQueueData_t::MAX_CAPACITY` we might break the port